packages/docs-site/src/library/pages/components/masthead.md

```diff
@@ -4,7 +4,17 @@ description: A top level section displayed at the top of the page
 header: true
```

Comment:
Are there some design docs that need to accompany this change? Maybe not?

Reply:
I will speak with @hxltrhuxze and raise a separate ticket if required.